tests/ParserSpecTest.php

```diff
@@ -32,6 +32,7 @@ public function specWithoutValueAndDefaultValueProvider()
             [new JsonParser($this->createExceptionFactory(), 'config', null), ['value' => true]],
             [new YesNoBooleanParser($this->createExceptionFactory(), 'isAwesome', null), true],
             [new BooleanParser($this->createExceptionFactory(), 'isNice', null), true]
+            // TODO: [spec, default-value] with comma-separated data types
```

Comment:
It's totally fine to keep them in another class, but then we should remove the todo :)